docs/constants.json

```diff
@@ -0,0 +1,6 @@
+{
+  "FORWARD_EVENT_WEBHOOK": {
+    "staging": "https://evj5gwoa8j.execute-api.us-east-1.amazonaws.com/hookshot/webhook",
```

Comment:
What PR did these land from and do they correctly belong in the repo?

Reply:
@asheemmamoowala this landed for the feedback component https://github.com/mapbox/mapbox-gl-js/pull/8507